### PR TITLE
Filter out large EXIF entries

### PR DESCRIPTION
**Pull request by artonge**

### CODE REVIEW

File: lib/Listener/ExifMetadataProvider.php [hunk 1]
```diff
@@ -169,7 +169,10 @@ private function sanitizeEntries(array $data): array {
 				$key = preg_replace('/[^a-zA-Z]/', '_', $key);
 			}
 
-			$cleanData[$key] = $value;
+			// Arbitrary limit to filter out large EXIF entries.
+			if (strlen($value) < 1000) {
+				$cleanData[$key] = $value;
+			}
```
Comment (ArtificialOwl) [inline lib/Listener/ExifMetadataProvider.php:175]: debug log entries that exceed this limit ? :)

**Review by ArtificialOwl — APPROVED** [review]

**Comment by ArtificialOwl** [comment]
why not filters on `UndefinedTag__x____` ?

File: lib/Listener/ExifMetadataProvider.php (hunk 1 above)
Comment (ArtificialOwl) [reply lib/Listener/ExifMetadataProvider.php:175]: might not need the value itself, just the key and the strlen() of the value ?

**Comment by artonge** [comment]
> why not filters on `UndefinedTag__x____` ?

Today it's `UndefinedTag__x____`, tomorrow, who knows? `UndefinedTag__x____` could also sometime be sane.

**Pull request merged**